bluepyopt/ephys/parameterscalers.py

```
@@ -110,12 +111,17 @@ def __init__(
                 object.
                 The distribution string should contain these names, and they
                 will be replaced by values of the corresponding attributes
+            soma_ref_location (float): location along the soma used as origin
+                from which to compute the distances. Expressed as a fraction
+                (between 0.0 and 1.0).
         """
 
         super(NrnSegmentSomaDistanceScaler, self).__init__(name, comment)
         self.distribution = distribution
 
         self.dist_param_names = dist_param_names
+        self.soma_ref_location = soma_ref_location
+        assert 0. <= self.soma_ref_location <= 1.
```

I think it's better to raise an Exception

---

(ValueError)